src/components/SuiteHeader/i18n/index.js

```diff
@@ -22,510 +22,8 @@ const SuiteHeaderI18N = {
-    about: 'Om',
-    support: 'Support',
-    gettingStarted: 'Komma igång',
-  },
-  tr: {
-    help: 'Yardım',
-    profileTitle: 'Profil',
-    profileManageButton: 'Profili yönetin',
-    profileLogoutButton: 'Oturumu kapat',
-    logout: 'Oturumu kapat',
-    userIcon: 'Kullanıcı',
-    administrationIcon: 'Yönetim',
-    settingsIcon: 'Ayarlar',
-    profileLogoutModalHeading: 'Oturumu kapatmak istiyor musunuz?',
-    profileLogoutModalSecondaryButton: 'İptal et',
-    profileLogoutModalPrimaryButton: 'Oturumu kapat',
-    profileLogoutModalBody: (solutionName, userName) =>
-      `${solutionName}'te ${userName} olarak oturum açtınız.  Oturumu kapattığınızda aynı tarayıcıda açık olan her bir uygulamadan çıkarsınız.  Güvenli bir oturum kapatma işlemi için tüm açık tarayıcı pencerelerini kapatın.`,
-    switcherNavigatorLink: 'Tüm uygulamalar',
-    switcherLearnMoreLink: 'Ek bilgi',
-    switcherRequestAccess: 'Contact your administrator to request application access',
-    whatsNew: 'Yenilikler',
-    documentation: 'Belge',
-    requestEnhancement: 'İstek geliştirmesi',
-    about: 'Hakkında',
-    support: 'Destek',
-    gettingStarted: 'Başlarken',
-  },
-  zh: {
-    help: '帮助',
-    profileTitle: '概要文件',
-    profileManageButton: '管理概要文件',
-    profileLogoutButton: '注销',
-    logout: '注销',
-    userIcon: '用户',
-    administrationIcon: '管理',
-    settingsIcon: '设置',
-    profileLogoutModalHeading: '要注销吗？',
-    profileLogoutModalSecondaryButton: '取消',
-    profileLogoutModalPrimaryButton: '注销',
-    profileLogoutModalBody: (solutionName, userName) =>
-      `您已经以 ${userName} 身份登录到 ${solutionName}。如果注销，也会从在同一浏览器中打开的每个应用程序中注销。为确保安全地注销，请关闭所有打开的浏览器窗口。`,
-    switcherNavigatorLink: '所有应用程序',
-    switcherLearnMoreLink: '了解更多信息',
-    switcherRequestAccess: 'Contact your administrator to request application access',
-    whatsNew: '新增内容',
-    documentation: '文档',
-    requestEnhancement: '请求增强',
-    about: '关于',
-    support: '支持',
-    gettingStarted: '入门',
-  },
-  'zh-TW': {
-    help: '說明',
-    profileTitle: '設定檔',
-    profileManageButton: '管理設定檔',
-    profileLogoutButton: '登出',
-    logout: '登出',
-    userIcon: '使用者',
-    administrationIcon: '管理',
-    settingsIcon: '設定',
-    profileLogoutModalHeading: '您要登出嗎？',
-    profileLogoutModalSecondaryButton: '登出',
-    profileLogoutModalPrimaryButton: '登出',
-    profileLogoutModalBody: (solutionName, userName) =>
-      `您已以 ${userName} 身分登入${solutionName}。如果登出，則也會將您登出您在同一瀏覽器中開啟的每個應用程式。若要確保安全登出，請關閉所有開啟的瀏覽器視窗。`,
-    switcherNavigatorLink: '所有應用程式',
-    switcherLearnMoreLink: '進一步瞭解',
-    switcherRequestAccess: 'Contact your administrator to request application access',
-    whatsNew: '新增功能',
-    documentation: '說明文件',
-    requestEnhancement: '要求加強功能',
-    about: '關於',
-    support: '支援',
-    gettingStarted: '開始使用',
+    surveyTitle: solutionName => `Enjoying ${solutionName}?`,
+    surveyText: 'Click here to help us improve the product',
```

Comment:
I've just got an update from id and the string is "To help us improve the product, **click here**.", with only the "click here" part being clickable. But I am ok with the way it is now.